### ISSUE TITLE
Enable users to update personal information

### ISSUE THREAD

**Issue by rantahar** [tystakartografen/cogs3#44]
#### Description
Users should be able to access a view for editing their profile and personal information.

#### Tests
Unit tests for the interaction between the form and the model
Integration test for accessing and submitting the form

**Comment by tystakartografen** [comment]
May need to wait until Phase 2 as this would require an additional API endpoint being created to update a users details in OpenLDAP.

**Comment by rantahar** [comment]
I was only thinking of the information we ask the user for and keep in the CustomUser table, like the first and last name. Are we planning to keep them in the database?

When we add email preferences, it should be possible to update those as well.

**Comment by tystakartografen** [comment]
Yes, basic user information is stored in the CustomUser table and the OpenLDAP service. I think any changes to the user's information in the CustomUser table needs to be propagated to the OpenLDAP service, which manages a user's access to the cluster.  Phase 1 has to be complete by Thursday (18 Jun 2018), I will try to get this pull request in for Phase 1.

**Comment by rantahar** [comment]
Do you mean Thursday 28th?
If this is not straightforward, would it make sense to merge this and create a separate issue about reflecting the changes in the OpenLDAP database?

**Comment by tystakartografen** [comment]
Oops, I meant the 28th (updated comment). Well, if I merge the pull request, then the functionality has to be supported in phase 1 and I'm not sure if I will have time to implement the OpenLDAP logic. I should have a better idea by the end of tomorrow if this pull request will make it into phase 1.
